pkg/storaged/dialog.jsx

```diff
@@ -1188,6 +1186,21 @@ function is_expected_unmount(usage, expect_single_unmount) {
             usage[0].usage == "mounted" && usage[0].location == expect_single_unmount);
 }
 
+const teardown_block_name = use => {
+    const block_stratis = client.blocks_stratis_fsys[use.block.path];
+    const block_btrfs = client.blocks_fsys_btrfs[use.block.path];
+    let name;
+    if (block_stratis) {
```

Comment:
Let's make it so that `use.name` has highest priority:

```
if (use.name)
  name = use.name;
else if (block_stratis)
 ...
else
  ...
```

Then we don't need to mention btrfs here.

Reply:
I don't think that works as we don't want to actually use `use.name`  at all. I created a follow up ticket to refactor this so all callers call the correct name